.github/workflows/pr-validation.yml

```
@@ -0,0 +1,132 @@
+
+              **GitHub issues:**
+              - PR title: "feat(api): GIT-123 Add new endpoint"
+              - PR description: "Resolves GIT-123" or "Fixes #123"
+              - Or link to the GitHub issue
+
+              This helps with tracking and project management. Thank you!`
+            })
+
+      - name: Check PR Description Completeness
+        id: check_description
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:5:28: Double quote to prevent globbing and word splitting

(shellcheck)